projects/user_management/resources/groups.tf

```diff
@@ -76,3 +81,35 @@ resource "aws_iam_policy" "base-user-console-access" {
     description = "base-user-console-access allows standard user tasks like create access key"
     policy = "${data.template_file.base-user-console-access.rendered}"
 }
+
+resource "aws_iam_policy" "ci_policy" {
+    name = "ci_policy"
+    description = "CI policy: lunch instances"
+    policy = <<EOF
```

Comment:
There is already a policy defined with a template and another one with a string, I'm trying to be consistent with what already have in place